Add package.json?

Would it be possible to add a package.json file (and possibly even publish to npm).
It'd make it slightly easier to install via npm. 

```sh
$ npm i Noitidart/ostypes -D

npm ERR! addLocal Could not install /var/folders/qc/g_f0szh95k1fzph34rhmf5jw0000gp/T/npm-37620-cbda1b4d/git-cache-ad8eb847/cba5d9fd34cea3dd89da31aa2e2ccbdf89a066bf
npm ERR! addLocal Could not install /var/folders/qc/g_f0szh95k1fzph34rhmf5jw0000gp/T/npm-37620-cbda1b4d/git-cache-674f1560/cba5d9fd34cea3dd89da31aa2e2ccbdf89a066bf
npm ERR! addLocal Could not install /var/folders/qc/g_f0szh95k1fzph34rhmf5jw0000gp/T/npm-37620-cbda1b4d/git-cache-e7b38fa8/cba5d9fd34cea3dd89da31aa2e2ccbdf89a066bf
npm ERR! Darwin 16.1.0
npm ERR! argv "/Users/pdehaan/.nvm/versions/node/v6.9.0/bin/node" "/Users/pdehaan/.nvm/versions/node/v6.9.0/bin/npm" "i" "Noitidart/ostypes" "-D"
npm ERR! node v6.9.0
npm ERR! npm  v3.10.8
npm ERR! code EISDIR
npm ERR! errno -21
npm ERR! syscall read

npm ERR! eisdir EISDIR: illegal operation on a directory, read
npm ERR! eisdir This is most likely not a problem with npm itself
npm ERR! eisdir and is related to npm not being able to find a package.json in
npm ERR! eisdir a package you are trying to install.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/pdehaan/dev/github/mozilla/min-vid/npm-debug.log
```

--- a/package.json
+++ b/package.json
@@ -1,0 +1,19 @@
+{
+  "name": "ostypes",
+  "description": "Submodule for Firefox addons that need js-ctypes to tap into platform (Windows, Mac, Linux/Unix) APIs.",
+  "version": "1.0.0",
+  "author": "Noitidart (http://noitidart.github.io/)",
+  "bugs": {
+    "url": "https://github.com/Noitidart/ostypes/issues"
+  },
+  "homepage": "https://github.com/Noitidart/ostypes#readme",
+  "keywords": [],
+  "license": "MPL-2.0",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Noitidart/ostypes.git"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  }
+}

